### PR TITLE
Ch7 issue15

### LINKED ISSUE #15: What are some questions to be asked to the project sponsor to understand their deployment requirements?
**Section Outline**
This section covers common considerations that engineers should clarify with sponsors in relation to:
- data transfer
- solution architecture
- system usage
- deployment testing/staging environment
- organisational restrictions

Note: To avoid questions regarding workflows & integration as these will be covered by issue #17.
 
**Definition of Done**
- [x] Completed `7-solution-delivery/deployment-requirements-gathering.md`
- [x] Verified against [self-review checklist](https://docs.google.com/document/d/1jIFbca53uD8b2FsDrUR6Dlb7k20SlZQBt-jpJ_qV14Y/edit)
- [x] Page builds and displays correctly

### PR DESCRIPTION
**Pull request by snjannah**

### CODE REVIEW

**Comment by chuawjk** [comment]
Looks good too! Thanks @snjannah!

**Comment by ngkh2018** [comment]
1. I would suggest to drop this portion (in the production environment’) from bullet point 1 under the ‘Data’ section, as we do not deal with production environment of the project sponsor in all our 100E projects.

2. Under the ‘Data’ section, model outputs are mentioned in bullet point 2, so I suggest to rename the section title from ‘Data’ to ‘Data/Model Outputs’.

**Comment by snjannah** [comment]
1. @ngkh2018 According to the contributor guidelines, the handbook content is not meant to be specific to 100E or AISG. Perhaps the term "deployment environment" would be better? It could mean any environment the solution is deployed to, be it test or production or others.

2. Thanks for the suggestion; I will rename accordingly.

**Comment by najibninaba** [comment]
The latest edits look good to me. Please proceed.

File: book/7-solution-delivery/deployment-requirements-gathering.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+3. Are there any restrictions or preferences regarding the operating system on which the solution should be deployed? 
+4. Would a staging environment or test device be provided for developers to test with prior to initial deployment? 
+    - If not provided, you would need to commission an internal environment that closely mirrors the specifications of the environment on which the solution would eventually be deployed.
+
+## Data
+1. Where would the data to be used for retraining and inference be located in the production environment? 
+2. Where would the model outputs be stored? Is there any storage size limit on this location? 
+3. If the data storage is located remotely, what is the authentication method to be used to access it? 
+
+## Usage
```
Comment (kevinosl) [inline book/7-solution-delivery/deployment-requirements-gathering.md:21]: Suggest adding a general comment in this section to say that often, the requirements and considerations for inference are more critical compared to retraining, as the latter is usually performed less frequently and is less time-sensitive.

Comment (kevinosl) [reply book/7-solution-delivery/deployment-requirements-gathering.md:21]: Another possible question under the General section is whether the sponsor has any restrictions on software licenses. We can call out that in general, we go for open source libraries under Apache 2.0, MIT and BSD licenses. If we want to use a less permissive license, we should check with the sponsor first.

Other than that, the article looks great Jannah.

**Comment by snjannah** [comment]
@ngkh2018 @kevinosl I have pushed my final draft to this branch based on your comments & suggestions.

**Comment by ryzalk** [comment]
Closing this pull request in view of the new merging workflow i.e. from contributing branches to `develop`.
CC @chuawjk

**Pull request closed without merging**